ci/ci_helper.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+""" Since github actions are quite difficult to use when writing scriptable things.
+We rely on this script being able to output the unaltered text.
+"""
+
+import os, sys, platform
+
+PLATFORMS_PRIVATE = ('x86_64-ps4', 'arm64-nx64')
+
+DIRNAME=os.path.dirname(os.path.abspath(__file__))
+
+# We use this function to determine if we should skip this platform
+def can_build_private_platform(platform):
+    return False
+
+def repo_name_to_platforms(repository):
+    return None
+
+if os.path.exists(os.path.join(DIRNAME, 'ci_private.py')):
+    import ci_private
+    can_build_private_platform = ci_private.can_build_private_platform
+    repo_name_to_platforms = ci_private.repo_name_to_platforms
```

Comment:
ON the private repos/branches, we'll add the `ci_private.py` with these functions